charts/kubeai/charts/models/values.yaml

```diff
@@ -51,29 +51,29 @@ catalog:
     features: ["TextEmbedding"]
     owner: nomic
     url: "ollama://nomic-embed-text"
-    server: OLlama
+    engine: OLlama
     resourceProfile: CPU:1
   # Opt #
   opt-125m-cpu:
     enabled: false
     features: ["TextGeneration"]
     owner: facebook
     url: "hf://facebook/opt-125m"
-    server: VLLM
+    engine: VLLM
     resourceProfile: CPU:1
   opt-125m-l4:
     enabled: false
     features: ["TextGeneration"]
     owner: facebook
     url: "hf://facebook/opt-125m"
-    server: VLLM
+    engine: VLLM
```

Comment:
I would just keep it server for now. I don't think engine is better. It might cause issues for some existing users this rename.

Reply:
Its actually already engine, I switched this commit reverted the change from `engine:` to `server:` that I introduced in the 1st commit of this PR. I agree we should not change this in this PR.

Reply:
Lol I see. I guess as long as we keep it as is, then it's good.